cdapython/utils/utility.py

```diff
@@ -101,23 +101,51 @@ def get_table_version() -> str:
 def unique_terms(
     col_name: str,
     system: str = "",
-) -> Q:
+    offset: int = 0,
+    host: Optional[str] = None,
+    verify: Optional[bool] = None,
+    async_req: Optional[bool] = True,
+    show_sql: bool = False,
+    show_counts: bool = False,
+    show_term_count: bool = False,
+    verbose: bool = True,
+    limit: int = 100,
+) -> Paged_Result:
     """
     Show all unique terms for a given column.
```

Comment:
The _description_ is empty on each of these.  Can you please explain what each are supposed to do?